.pre-commit-config.yaml

```diff
@@ -0,0 +1,41 @@
+repos:
+  - repo: https://github.com/PyCQA/flake8
+    rev: 6.0.0
+    hooks:
+      - id: flake8
+        additional_dependencies:
+          - flake8-executable
+          - bandit
+          - flake8-bugbear
+          - flake8-comprehensions
+          - flake8-simplify
+        args: [ "--max-line-length", "120" ]
+        exclude: "tests/to-be-tested/"
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: 'v0.991'
+    hooks:
+      - id: mypy
+        args: ["--disallow-untyped-defs", "--show-error-codes"]
```

Comment:
Would we be able to keep `.mypy.ini` and remove this argument list?

Reply:
I'm not sure whether this is the best practice, but I saw you have replaced other configurations with args as well. Is this just a personal preference, or do you have any further reasons?

Reply:
Configuration for python tools is moving towards `pyproject.toml`, where not every tool needs another file at the root of the repository. Specifying the args directly in `pre-commit` also provides overview. I'm fine with both, would you prefer `.toml`?

Reply:
Ok if we move the config to `pyproject.toml` in the next PR?

Reply:
Yes, I agree with should move to `pyproject.toml` in the next PR. The project started at a time when `pyproject.toml` wasn't widely supported. I totally agree with changing to that direction (in a separate PR)

Reply:
The CI should also pass once you restore those configuration file in this PR